### PR TITLE
Hannes ffl/fin 124 update profile route

### PR DESCRIPTION
**Pull request by hnns-dev**
#124 update profile route

### CODE REVIEW

File: app/api/profile/user/route.ts
Comment (dnmct) [inline app/api/profile/user/route.ts:1]: is this file used? if yes make it a server action

**Review by dnmct — DISMISSED** [review]

File: seed.ts [hunk 1]
```diff
@@ -25,7 +25,7 @@ async function main() {
   });
 
   const musselGym = await prisma.venue.upsert({
```
Comment (louziferr) [inline seed.ts]: Upsert only works when the Venue already exists. Please user prisma.venue.create instead. Or just use the seed file from main branch.

File: app/profile/me/page.tsx [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import { redirect } from "next/navigation";
+
+const MePage = async () => {
+  // const user = await protectPage()
+  const ownUserId = "jhvrihll2lalvsug";
```
Comment (louziferr) [inline app/profile/me/page.tsx]: Please use the actual ID from the currently logged in user.

File: app/profile/page.tsx
Comment (louziferr) [inline app/profile/page.tsx:1]: This page shouldn't exists or only exist with a request parameter that is a user's ID. Please make responsive and request the profile page of an actual user from the DB, using their ID.

**Review by louziferr — APPROVED** [review]
very clean and cool! Nice job :)

**Pull request merged**